packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -67,7 +74,12 @@ export interface ClonableSchemaAndPolicy extends SchemaAndPolicy {
  * TODO: actually implement
  * TODO: is history policy a detail of what indexes are used, or is there something else to it?
  */
-export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends SharedObject {
+export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange>
+	extends SharedObject
+	implements WithBreakable
```

Comment:
I wonder if the `SharedTree` class should also have some methods (e.g., `schematizeFlexTree` and `viewWith`) decorated with `@throwIfBroken`. You're probably in a better position than I to determine that, so I'm happy to follow your lead here.

Reply:
Yes, I think so. No attempt was made to be complete here, just establish a pattern.